README.md

```diff
@@ -1 +1,52 @@
-# Microverse-Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse-Portfolio
+
+Portfolio: setup and mobile version skeleton
+In this project, I set up a "My portfolio: mobile version" repository.
+
+## Built With
+
+- HTML & CSS
+- Flex-box
+
+## Getting Started
+
+node_modules were ignored by .gitignore file
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
- [ ] [OPTIONAL] Kindly ensure you add information on how to get started with this project, prerequisites,setup, install and usage.